src/components/UserManagement/UserManagement.tsx

```diff
@@ -49,16 +53,18 @@ const UserManagement = ({
   return (
     <div {...commonProps} className={classnames(baseClassName, className)} {...props}>
       <ul className={`${baseClassName}__account-wrapper`}>
-        {isLoggedIn && (
-          <AccountDetailsComponent>
-            <AccountCircle className={`${baseClassName}__account-icon`} />
-          </AccountDetailsComponent>
+        {authState !== AuthState.Loading && (
```

Comment:
I always liked pulling these conditional checks out into their own variables. i.e. 
`const showAccountDetails = authState !== AuthState.Loading;`
Not a huge deal here, but it does make things a little easier to maintain down the line if new conditions ever get added.

Reply:
good idea!